com.valvesoftware.Steam.yml

```diff
@@ -35,8 +35,10 @@ finish-args:
   - --talk-name=org.freedesktop.Notifications
   - --filesystem=xdg-music:ro
   - --filesystem=xdg-pictures:ro
+  - --filesystem=xdg-run/mumble:create
   - --filesystem=xdg-run/app/com.discordapp.Discord:create
   - --device=all
+  - --device=shm
```

Comment:
I suppose this conflicts with `--allow=per-app-dev-shm`
Also, given this permission is pretty unsafe (many apps use `/dev/shm` for IPC), are you sure it's strictly required for the overlay to work? If it communicates via a socket in `$XDG_RUNTIME_DIR`, what does it need `/dev/shm` for?

Reply:
Okay, I investigated a little and found that the shared memory is used by the overlay to retrieve the image rendered by the Mumble client. We are talking about ~3.7MB of data in shared memory.
As far as I know that is read at least (if not more) every time the overlay and the resolution change.

Metadata for that image is sent through a local socket. I am now trying to understand if it's feasible and a good idea (performance-wise) to move everything to the socket. That includes being able to use the overlay in multiple games, simultaneously